### ISSUE TITLE
eslint-plugin-depend should not flag eslint-plugin-unicorn

### ISSUE THREAD

**Issue by the-ult** [es-tooling/eslint-plugin-depend#49]
```
"eslint-plugin-unicorn" should be replaced with an alternative package. Read more here: https://github.com/es-tooling/module-replacements/blob/main/docs/modules/eslint-plugin-unicorn.mdeslint[depend/ban-dependencies](https://github.com/es-tooling/eslint-plugin-depend/blob/main/docs/rules/ban-dependencies.md)
```

It is kind of strange that this repo/plugin is flagging the original eslint-plugin-unicorn, to favor it's own eslint-plugin-unicorn-x. While the original is still being maintained.
And it seems from the discussions, that there is just a different opionion on some manners. But this does not seem to be a valid reason to flag it dat is should be replaced. Just because the maintainer of es-tooling/eslint-plugin-depend has a different opinion and is therefore flagging the original. Correct?

See: https://github.com/es-tooling/eslint-plugin-unicorn-x/issues/6

cc @fisker @sindresorhus [43081j](https://github.com/43081j)

**Comment by 43081j** [comment]
I've seen the commotion and confusion so I will explain this better soon and make a decision of if to keep it in here or not 👍

**Comment by fisker** [comment]
Unsubscribing, since I don't really care. 😄

**Comment by 43081j** [comment]
ok cool, all sorted

TLDR - it was a pre-emptive move adding unicorn-x to the replacements repo and this plugin. i have now reverted that until the fork becomes more widely used (if ever).

on why there is a fork, it doesn't really matter if the original is maintained or not. the fork exists because we wanted to make performance changes, some of which will be (or have been) rejected upstream. there's no movement on some of these, and we are going to make the changes, so the only way forward is a fork.

that is fine and is part of why forking is possible in OSS. we want it to behave differently than the source project, in a way that the source project doesn't want.

however, we shouldn't be recommending one over the other at this point. the fork is much faster and lighter, but the original repo is still active and well maintained.

a large amount of work has gone into making the fork faster, a lot of debugging of rules and digging through CPU profiles, etc. it does indeed have reason to exist, but doesn't have reason to entirely replace the original right now.
